### PR TITLE
[TASK] Add .gitattributes file to exclude files for composer installations

### PR DESCRIPTION
**Pull request by simonschaufi**
Exclude those files in zip archive whenever the generated extension is fetched via composer.

### CODE REVIEW

File: Resources/Private/CodeTemplates/Extbase/gitattributes.t [hunk 1]
```diff
@@ -0,0 +1,9 @@
+/Documentation export-ignore
+/Tests export-ignore
+/.editorconfig export-ignore
+/.gitattributes export-ignore
+/.gitignore export-ignore
+/.php_cs export-ignore
+/.scrutinizer.yml export-ignore
+/.travis.yml export-ignore
+/ExtensionBuilder.json
```
Comment (nicodh) [inline Resources/Private/CodeTemplates/Extbase/gitattributes.t]: I don't understand why you add ExtensionBuilder.json here?

Comment (simonschaufi) [reply Resources/Private/CodeTemplates/Extbase/gitattributes.t]: Ups, I forgot the export-ignore in this line

Comment (nicodh) [reply Resources/Private/CodeTemplates/Extbase/gitattributes.t]: The whole extension i only needed in development...
No need to install it in LIVE installations...
??

Comment (simonschaufi) [reply Resources/Private/CodeTemplates/Extbase/gitattributes.t]: This is not for the extension_builder extension but for the generated extension code. I guess you missed that point ;)

**Pull request merged**